src/actions/index.js

```diff
@@ -0,0 +1,76 @@
+export const login = userCredentials => {
+  return {
+    type: 'LOGIN',
+    userCredentials
+  }
+}
+
+export const projects = () => {
+  return {
+    type: 'PROJECT'
+  }
+}
```

Comment:
It should be renamed like some action keyword, like `getProjects` & type as **GET_PROJECTS**